CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 FEATURES:
 
 * **New Resource:** `cloudflare_waiting_room_event` ([#1509](https://github.com/cloudflare/terraform-provider-cloudflare/issues/1509))
+* **New Resource:** `cloudflare_teams_proxy_endpoint`
```

Comment:
this file is automatically generated, check out https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/docs/changelog-process.md to add a changelog entry.

Reply:
fixed!